src/accessories/SearchAssistant.ttslua

```diff
@@ -130,7 +135,36 @@ function startSearch(messageColor, number)
   setAsideRotation = { handData.rotation.x, handData.rotation.y + 180, 180 }
 
   -- place hand cards set aside
-  deckLib.placeOrMergeIntoDeck(handCards, setAsidePosition, setAsideRotation)
+  if #handCards > 0 then
+    deckLib.placeOrMergeIntoDeck(handCards, setAsidePosition, setAsideRotation)
+  else
+    oldSnapPoints = Global.getSnapPoints()
+    newSnapPoints = oldSnapPoints
```

Comment:
I don't think this works because Lua passes tables by reference, not by value

Reply:
yeah, see my EDIT comments above -- I just left this in here as a reminder to me/you to figure out how to make this work ;)